extensions/attention-stream/src/youtube/injector.content.ts

```diff
@@ -0,0 +1,17 @@
+export {};
+
+// Inject the monkeypatch script into YouTube pages
+(function() {
```

Comment:
Same as above, instead of doing this from an "injector" content script, I think you could get the tab ID from the `sender` on the content script message and inject into that tab using [this API](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/scripting/executeScript).

Reply:
(replied above)